### ISSUE TITLE
Built container could not be found during tests

### ISSUE THREAD

**Issue by dweidenfeld** [testcontainers/testcontainers-node#21]
Hi,

I am currently facing the issue, that my freshly built container could not be found to start it for the tests.

```js
const appPort: number = 3000

let appImage: GenericContainer = new GenericContainer(image)
if (!image) {
  appImage = await GenericContainer.fromDockerfile(path.resolve(__dirname, '..'))
}

const bully: StartedTestContainer = await appImage.withExposedPorts(appPort).start()
```

This fails with:
```
pull access denied for 5668e0a8f7a4fb86f08d814c41dc4402, repository does not exist or may require 'docker login'
```

The uuid `5668e0a8f7a4fb86f08d814c41dc4402` is the generated one.

Does somebody else had those issues before?
Is there a solution for it?

**Comment by cristianrgreco** [comment]
Hi

I haven't seen that issue before.

Could there be something in your Dockerfile which is pulling from a private repo?

**Comment by dweidenfeld** [comment]
Actually yes. But is there an option to pass build-args?

**Comment by cristianrgreco** [comment]
Not as of yet

**Comment by dweidenfeld** [comment]
Okay, I'll open a PR if I can make it work ;)

**Comment by cristianrgreco** [comment]
Your PR has been merged and released in v2.0.0. Please give it a go and report back
